Sources/Data/Services/ExperienceStore/ExperienceStoreService.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ExperienceStoreService.swift
+//  RoverData
+//
+//  Created by Sean Rucker on 2018-05-03.
+//  Copyright © 2018 Rover Labs Inc. All rights reserved.
+//
+
+import Foundation
+import os
+
+class ExperienceStoreService: ExperienceStore {
+    func get(byID id: String) -> Experience? {
+        do {
+            guard let path = experienceLocalURL(id: id) else {
+                os_log("Not able to retrieve experience because local storage for it could not be determined.")
+                return nil
+            }
+            let json = try Data.init(contentsOf: path)
+            return try JSONDecoder.default.decode(Experience.self, from: json)
+        } catch {
+            os_log("Unable to retrieve experience locally: %s", log: .persistence, type: .error, error.localizedDescription)
+            return nil
+        }
+    }
+    
+    func insert(experience: Experience) {
```

Comment:
Perhaps this method should throw rather than swallow errors... this would allow the calling instance to determine how they want to handle errors. As it is, the caller has no idea if this succeeded or not. Another option would be to return a bool?